xml/operations-compute-using_flavor_metadata.xml

```diff
@@ -8,143 +8,29 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic_vhs_12v_vw">
  <title>Using Flavor Metadata to Specify CPU Model</title>
  <para>
-  In &kw-hos; Nova, the <literal>ComputeCapabilitiesFilter</literal>
+  In &kw-hos; &o_comp;, the <literal>ComputeCapabilitiesFilter</literal>
   scheduling filter matches the exact CPU model of the compute host and the
-  requested compute instance model. It doesn't entertain the libvirt's
-  capability of emulating the hosts CPU model to its guests if the guest's CPU
-  model is older or equal to that of the hosts.
+  requested compute instance model. Libvirt supports emulating guest CPUs that
+  are different from the host CPU. However, &o_comp; does not support that
+  feature if the guest's CPU model is older or equal to that of the hosts.
```

Comment:
I don't think this new version makes sense -- I find it rather unlikely that you could emulate a *newer* processor than what you actually have. That would be a pretty extreme case of forward compatibility.